tests/fxl-layout_duplication/OPS/package.opf

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<package xmlns="http://www.idpf.org/2007/opf" xmlns:epub="http://www.idpf.org/2007/ops" version="3.0" xml:lang="en" unique-identifier="pub-id">
+  <metadata xmlns:dc="http://purl.org/dc/elements/1.1/">
+    <dc:coverage>Fixed Layout</dc:coverage>
+    <dc:creator>Wendy Reid</dc:creator>
+    <dc:date>2021-11-24</dc:date>
+    <dc:description>
+      When rendition:layout is duplicated, EPUBCheck must return an error.
+    </dc:description>
+    <dc:identifier id="pub-id">fxl-layout_duplication</dc:identifier>
```

Comment:
As noted in the epub-specs PR, there should only be an underscore here if the ID would *otherwise* be fxl-layout, but you have multiple tests for the same requirement and so are using the underscore to distinguish them.

Below, the anchor appears to be fxl-layout-duplication, so I think the ID, and the filenames, should be fxl-layout-duplication.

Same idea applies throughout the PR.